test/getTransferSpec.js

```diff
@@ -132,8 +133,28 @@ describe('GET /transfers/:execution_condition/execution_condition', function ()
     yield this.request()
       .get(`/transfers/${transfer.execution_condition}/execution_condition`)
       .expect(200)
-      .expect(transfer)
-      .expect(validator.validateTransfer)
+      .expect([transfer])
+      .expect(validator.validateTransfers)
+      .end()
+  })
+
+  it('should return 200 for multiple existing transfers', function * () {
+    yield dbHelper.addTransfers([this.preparedTransfer])
+
+    const transfer = this.executedTransfer
+    yield this.request()
+      .get(`/transfers/${transfer.execution_condition}/execution_condition`)
+      .expect(200)
+      .expect((res) => {
+        const sortedResponse = _.sortBy(res.body, (transfer) => {
```

Comment:
Not necessarily. It depends on the order they are returned from the database since we aren't doing any sorting.